src/aux-vm-browser/vm/AuxVMImpl.ts

```diff
@@ -237,6 +238,32 @@ export class AuxVMImpl implements AuxVM {
      */
     async sendEvents(events: BotAction[]): Promise<void> {
         if (!this._proxy) return null;
+        if (events && events.length) {
+            const transferables = [];
+            for (let event of events) {
+                if ((event as any).result) {
```

Comment:
@Blitzy `BotAction` is a discriminated union based on `type`, so you should just be able to check if `event.type === 'async_result'`